### PR TITLE
feat(alert): Create metric alerts on project creation

### PR DESCRIPTION
**Pull request by PixelDust22**
image

Added a Performance Alerts section to the "Create a new Project" page. The selections currently mirror the preset configs except without the "target the highest volume transaction" part

### CODE REVIEW

File: static/app/views/projectInstall/createProject.tsx [hunk 1]
```diff
@@ -214,6 +217,45 @@ class CreateProject extends Component<Props, State> {
         );
         ruleId = ruleData.id;
       }
+      if (metricAlertPresets && metricAlertPresets.length > 0) {
+        const presets = PRESET_AGGREGATES.filter(aggregate =>
+          metricAlertPresets.includes(aggregate.id)
+        );
+        const teamObj = this.props.teams.find(aTeam => aTeam.slug === team);
+        for (const preset of presets) {
```
Comment (scefali) [inline static/app/views/projectInstall/createProject.tsx:225]: @Neo-Zhixing should we make these API requests in parallel with `Promise.all`?

Comment (Neo-Zhixing) [reply static/app/views/projectInstall/createProject.tsx]: Fixed

File: static/app/utils/analytics/workflowAnalyticsEvents.tsx [hunk 2]
```diff
@@ -52,6 +52,7 @@ export type TeamInsightsEventParameters = {
     session_id: string;
     wizard_v3: string;
   };
+  'project.create_page_viewed': {};
```
Comment (scefali) [inline static/app/utils/analytics/workflowAnalyticsEvents.tsx:55]: @Neo-Zhixing Maybe we should call this `project_creation_page.viewed` so we can group other events on that page with the same prefix.

File: static/app/utils/analytics/workflowAnalyticsEvents.tsx [hunk 3]
```diff
@@ -75,4 +76,5 @@ export const workflowEventMap: Record<TeamInsightsEventKey, string | null> = {
   'issue_details.viewed': 'Issue Details: Viewed',
   'new_alert_rule.viewed': 'New Alert Rule: Viewed',
   'team_insights.viewed': 'Team Insights: Viewed',
+  'project.create_page_viewed': 'Project Create: Creation page viewed',
```
Comment (scefali) [inline static/app/utils/analytics/workflowAnalyticsEvents.tsx:79]: @Neo-Zhixing the Amplitude name doesn't quite match the reload name as well

**Review by scttcper — APPROVED** [review]
looks good, i am worried we'll break the alert format being sent via POST and not know

**Pull request merged**